src/set.js

```diff
@@ -0,0 +1,71 @@
+export default class Set {
+  constructor() {
+    this.data = [];
+    this.length = 0;
+  }
+
+  add( stuff ) {
+      if( this.data.length > 0 ){
+        for( var i = 0; i < this.data.length; i++ ){
+          if( this.data[i] == stuff ){
+            return "ERROR, " + stuff + " already exists"
```

Comment:
indentation and consistant spacing. 
also this simple if/else could be a ternary.